packages/graphql-lowcode/src/modify/addFragmentFields.ts

```diff
@@ -0,0 +1,72 @@
+import { graphql, parse, visit } from 'graphql'
+
+export function addFragmentField(graphqlQuery: string, options: { entity: string, property: string }) {
+  //if fragment does not contain fragment keyword -> returns unchanged query
+  if (!graphqlQuery.indexOf('fragment')) return graphqlQuery
+
+  const ast = parse(graphqlQuery)
+  let finalQuery = graphqlQuery
```

Comment:
move this variable close to the place where it is used for the first time
initialize with an empty string (see the reason in comment below)